packages/atomic/src/pages/accessibility/commerce-full.html

```diff
@@ -69,6 +69,36 @@
         paginationSelect.addEventListener('change', (e) =>
           pagination.classList.toggle(useLoadMoreResultsClass, e.target.value === 'load-more')
         );
+
+        searchInterface.i18n.addResourceBundle('en', 'translation', {
+          'no-ratings-available': 'No ratings available',
+        });
+
+        document.querySelector('atomic-search-box-instant-results').ariaLabelGenerator = ({i18n}, result) => {
+          const information = [result.title];
+
+          if ('ec_rating' in result.raw) {
+            information.push(
+              i18n.t('stars', {
+                count: result.raw.ec_rating,
+                max: 5,
+              })
+            );
+          } else {
+            information.push(i18n.t('no-ratings-available'));
+          }
+
+          if ('ec_price' in result.raw) {
+            information.push(
+              result.raw.ec_price.toLocaleString(i18n.languages, {
+                style: 'currency',
+                currency: 'USD',
+              })
+            );
+          }
+
+          return information.join(', ');
```

Comment:
Renders the following labels in the DOM:
* `"Apple Pencil (1st Generation), 4.7 stars out of 5, $99.99, instant result. Button. 1 of 5. In Right list."`
* `"Sport Band for Apple Watch™ 44mm, No ratings available, $49.00, instant result. Button. 3 of 5. In Right list."`